smart-contracts/contracts/CosmosBridgeStorage.sol

```diff
@@ -58,8 +59,13 @@ contract CosmosBridgeStorage {
         uint256 amount;
     }
 
+    /**
+    * @notice network descriptor
+    */
+    uint256 public networkDescriptor;
```

Comment:
Again, let's test an upgrade off a mainnnet fork with the proper oz config file loaded up and ensure that the framework plays nice with this change.